frontend/src/component/splash/SplashOverlay/ReleaseManagementSplash.tsx

```diff
@@ -159,32 +204,38 @@ export const ReleaseManagementSplash = ({
                         <OpenInNewIcon fontSize='small' />
                         View documentation
                     </StyledLink>
-                    <StyledLink
-                        href={RELEASE_NOTES_URL}
-                        target='_blank'
-                        rel='noopener noreferrer'
-                    >
-                        <ArticleIcon fontSize='small' />
-                        Release notes
-                    </StyledLink>
                 </LinksRow>
 
-                <YouTubeContainer>
-                    <iframe
-                        src={`https://www.youtube.com/embed/${YOUTUBE_VIDEO_ID}`}
-                        title='Release management introduction video'
-                        allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
-                        allowFullScreen
-                    />
-                </YouTubeContainer>
+                <VideoContainer>
+                    {isPlaying ? (
+                        <iframe
+                            src={`https://www.youtube.com/embed/${YOUTUBE_VIDEO_ID}?autoplay=1`}
+                            title='Release management introduction video'
+                            allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
+                            allowFullScreen
+                        />
+                    ) : (
+                        <>
+                            <VideoThumbnail
+                                src={`https://img.youtube.com/vi/${YOUTUBE_VIDEO_ID}/hqdefault.jpg`}
+                                alt='Video thumbnail'
+                            />
+                            <PlayOverlay onClick={handlePlayClick}>
+                                <PlayButton>
+                                    <PlayArrowIcon />
+                                </PlayButton>
+                            </PlayOverlay>
+                        </>
+                    )}
+                </VideoContainer>
 
                 <ActionsRow>
                     <Button
                         variant='contained'
                         color='primary'
                         onClick={handleGetStarted}
                     >
-                        Get started with release management
+                        View the getting started guide
```

Comment:
It says it navigate to a getting started **guide**, but it navigates to release templates page.